js/layout-javascript/small-card-code.js

```diff
@@ -1623,6 +1646,14 @@ DynamicList.prototype.addDetailViewData = function (entry) {
       content = entry.originalData[dynamicDataObj.column];
     }
 
+    if (dynamicDataObj.type == 'image') {
+      content = entry.originalData[dynamicDataObj.column].split(/\n/);
+
+      content.forEach(function(imageUrl) {
```

Comment:
Can we not use `.map` here?

Reply:
@tonytlwu 
We actually need "content" variable. We use it later. Therefore we can use map instead of forEach, but I do not see any benefit of using it.